Functionality_Tests/FunctionalityTests.cs

```diff
@@ -0,0 +1,63 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
+using SeleniumExtras.WaitHelpers;
+using OpenQA.Selenium.DevTools.V115.Page;
+
+namespace Functionality_Tests_Suit
+{
+    [TestFixture]
+
+    public class FunctionalityTests : BaseTest
+    {
+
+        [Test, Order(1)]
+
+        public void PriceSorting()
+        {
+            SuccessfulLogin();
+            var elementItemSorting = driver.FindElement(By.TagName("select"));
+            elementItemSorting.Click();
+            var elementPriceSorting = driver.FindElement(By.CssSelector("[value='lohi']"));
+            elementPriceSorting.Click();
+            var elementMinPrice = driver.FindElement(By.CssSelector("[class='inventory_list']:first-child"));
+            Assert.That(elementMinPrice.Text.Contains("7.99"));
+
+            var elementItemsList = driver.FindElements(By.ClassName("inventory_list"));
+            Assert.That(elementItemsList, Is.Ordered.Ascending);
+        }
+    
+        [Test, Order(2)]
+
+        public void RemoveFromCart()
+        {
+            AddProductToCart();
+            var elementShopingcartIcon = driver.FindElement(By.ClassName("shopping_cart_link"));
+            elementShopingcartIcon.Click();
```

Comment:
Please add check that after clicking the "Add to Cart" button for the selected product the cart icon in the top-right corner displays the correct number of items added (e.g., 1 item).